.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,60 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main ]
+  schedule:
+    - cron: '17 23 * * 3' # run at 08:17 AM Thursdays, JST
```

Comment:
By the way, why `08:17 AM`? :thinking:

Reply:
I don't know why, but it's the time listed in the template :confused:

Reply:
Okay, then let's change it as we like!